src/home/NewsFeed/NewsFeed.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import { PreviewCard } from '../PreviewCard'
+
+export const NewsFeed = ({submissions}) => submissions.map(submission => <PreviewCard {...submission} />);
```

Comment:
NIT: Submissions will likely have an `_id` we can use as the react list id prop.